src/xrPhysics/xrPhysics.cpp

```diff
@@ -24,7 +24,7 @@ static void load(int argc, char** argv, char** envp)
 }
 
 #if defined(XR_PLATFORM_WINDOWS)
-BOOL APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserved)
+BOOL APIENTRY DllMain(HMODULE hModule, u32 ul_reason_for_call, LPVOID lpReserved)
```

Comment:
WinAPI function, should use WinAPI types.
```suggestion
BOOL APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserved)
```